### ISSUE TITLE
Failed to detect all players start out alive

### ISSUE THREAD

**Issue by rorosaurus** [denverquane/amonguscapture#14]
I created a lobby, with 4 other accounts, and started the game. All the muting worked as expected, so I voted the imposter (Mr. Meme). It appears as though the capture program noticed the game ended (state change to lobby), then labelled Mr. Meme as dead.

I started a new game. The capture program still thinks Mr. Meme is dead, and treats them as such, leaving them unmuted at the very beginning of the game (and muting them during discussion).

Can we change the capture program to invalidate/update any cache of each player's state when we transition *from* LOBBY (game start)?

````
Connected to Among Us process (19328)
(2014511104)
Mr Meme: Joined
connect 059A7F
State changed to LOBBY
Mr Meme: ForceUpdated
Mr Meme 1: Joined
Mr Meme 2: Joined
Mr Meme 3: Joined
Mr Meme 4: Joined
State changed to TASKS
State changed to DISCUSSION
Mr Meme: Exiled
State changed to LOBBY
Mr Meme: Died
Mr Meme 1: Left
Mr Meme 2: Left
Mr Meme 3: Left
Mr Meme 4: Left
Mr Meme 1: Joined
Mr Meme 2: Joined
Mr Meme 3: Joined
Mr Meme 4: Joined
State changed to TASKS
dState changed to DISCUSSION
State changed to TASKS
````

**Comment by denverquane** [comment]
Hey thanks a ton for posting logs!!
It looks like you're right; the died event is fired after getting back to the lobby.

However, I think from an implementation perspective, it might actually be easier to filter this on the Discord side, not the capture side. I think in 3 ways; the discord side should handle the "EXILE" event, it should never flag players as dead when in the lobby phase, and finally, it might as well mark all players as alive when going from Lobby -> Tasks

Appreciate you posting details for this!

**Comment by Aasikki** [comment]
Hmm this might be the same issue I was experiencing [here](https://github.com/denverquane/amongusdiscord/issues/53)

**Comment by denverquane** [comment]
Should be resolved in the latest (2.0.5+) versions of the discord exe, but reopen if you see this issue again
